microservices-modules/helidon/helidon-se/pom.xml

```diff
@@ -28,13 +28,13 @@
         <dependency>
             <groupId>io.helidon.webserver</groupId>
             <artifactId>helidon-webserver-netty</artifactId>
-            <version>${helidon.version}</version>
```

Comment:
let's keep using property placeholders to easily control version

Reply:
DONE